Zend/zend_inheritance.c

```diff
@@ -1750,6 +1750,59 @@ static uint32_t zend_check_trait_usage(zend_class_entry *ce, zend_class_entry *t
 }
 /* }}} */
 
+/** {{{ zend_warn_ambiguous_trait_methods
+ * Emits a warning when binding multiple Traits to a PHP Class will yield an ambiguous method.
+ * 
+ * This condition will become a fatal error in PHP 8.
+ * 
+ * @param zend_class_entry* ce A PHP Class undergoing compilation by the Zend engine that contains Traits which will be bound to the Class.
+ * @param zend_class_entry** traits An array of pointers to all Traits being bound to the given Class.
+ * @param int trait_index The index of the Trait we're currently binding to the given Class.
+ * @see https://www.php.net/manual/en/migration80.incompatible.php#migration80.incompatible.core.other
+ */
+static void zend_warn_ambiguous_trait_methods(zend_class_entry *ce, zend_class_entry **traits, int trait_index)
+{
+	// If the current Trait method is bound to a class, nothing more to do here
+	size_t i = trait_index, j = 0;
```

Comment:
it seems unnecessary to declare `i` here, it seems like you could just use `trait_index` directly on the class entry pointer